### PR TITLE
adding EnabledMfas to cognito UserPool Object.

### PR DESCRIPTION
**Pull request by castaway2000**
related to this https://github.com/cloudtools/troposphere/issues/1506

### CODE REVIEW

File: troposphere/cognito.py [hunk 1]
```diff
@@ -205,6 +205,7 @@ class UserPool(AWSObject):
         'EmailVerificationMessage': (basestring, False),
         'EmailVerificationSubject': (basestring, False),
         'LambdaConfig': (LambdaConfig, False),
+        'EnabledMfas': (basestring, False),
```
Comment (markpeek) [inline troposphere/cognito.py]: Minor fixup, please move this line up one to alphabetize the properties. Thanks!

Comment (castaway2000) [reply troposphere/cognito.py]: updated. thanks.

**Comment by markpeek** [comment]
Thanks!

**Comment by markpeek** [comment]
@castaway2000 I looked at #1508 and the new resources available. I was going to add them via my generator script. However, when looking at EnabledMfas, I missed that it is a list of strings. Can you confirm your usage of it and then I can correct with my other commit.

**Comment by castaway2000** [comment]
@markpeek my implementation is direct from the CFT docs where it is a list of strings. 

Userpool doc: 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cognito-userpool.html

direct doc:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cognito-userpool.html#cfn-cognito-userpool-enabledmfas

i am building with the latest revision of troposphere but edited and ran the CFT confirming the current implementation to be working. then copied the code over directly for this PR. 

let me know if you need any additional info.

**Comment by markpeek** [comment]
OK, then instead of this `'EnabledMfas': (basestring, False),` it should be implemented `'EnabledMfas': ([basestring], False),`. Note the brackets that imply a list of strings.

**Comment by castaway2000** [comment]
@markpeek as soon as I realized it i made a PR for it. https://github.com/cloudtools/troposphere/pull/1511

**Pull request merged**